pkg/util/crds.go

```diff
@@ -17,6 +17,11 @@ import (
 )
 
 func EnsureCRD(ctx context.Context, config *rest.Config, manifest []byte, groupVersionKind schema.GroupVersionKind) error {
+	// TODO: find a better way to deal with this
+	if config.Host == "unit-test-client" {
+		return nil
+	}
+
```

Comment:
could we have a `func` variable inside the util package that we can replace at runtime with the fake manager? 

something like 
```go
var ensureCrd func(ctx context.Context, config *rest.Config, manifest []byte, groupVersionKind schema.GroupVersionKind) error {
      create/update the crd
}

func UnitTestSetup(){
   ensureCRD = func(...)error{
    return nil
   }
}
```

Reply:
Oh yeah good idea, let me do that